app/controllers/foreman_bootdisk/api/v2/disks_controller.rb

```diff
@@ -10,7 +10,6 @@ class DisksController < ::Api::V2::BaseController
           api_base_url "/bootdisk/api"
         end
 
-        before_filter :find_resource, :only => :host
```

Comment:
> To summarise, you want to move the subnet action from v2/disks_controller into new controller like v2/subnet_disks_controller?

Correct.



> Ok, since this would require breaking existing route ('/hosts/:host_id') I will do that in a separate pull request.

No need to change the existing actions or controller at the moment, they work as-is and could be renamed later.